Library/Homebrew/cmd/uses.rb

```diff
@@ -77,24 +82,35 @@ def uses
 
   def intersection_of_dependents(use_runtime_dependents, used_formulae, args:)
     recursive = args.recursive?
+    include_formulae, include_casks = if !args.formula? && !args.cask?
```

Comment:
In most other commands this distinction is called `only` (`nil`, `:formula` or `:cask`). Would be good to follow the same pattern here.

Reply:
Oh yeah, much cleaner. Thanks!